openassessment/assessment/api/peer.py

```diff
@@ -24,7 +24,7 @@
 PEER_TYPE = "PE"
 
 
-def can_be_skipped(submission_uuid, peer_requirements):
+def can_be_skipped(submission_uuid, peer_requirements):  # pylint: disable=unused-argument
```

Comment:
Why is this? Can you remove the argument if it's not used?

Reply:
I've just tried to keep consistency on API calls. I am calling it from [here](https://github.com/open-craft/edx-ora2/blob/shimulch/bb-2732-make-peer-dependent-steps-skipable/openassessment/workflow/models.py#L844). I kept same functiontion signature as seen as [here](https://github.com/open-craft/edx-ora2/blob/shimulch/bb-2732-make-peer-dependent-steps-skipable/openassessment/workflow/models.py#L913) and [here](https://github.com/open-craft/edx-ora2/blob/shimulch/bb-2732-make-peer-dependent-steps-skipable/openassessment/workflow/models.py#L918).

Previously I was using those arguments to check if there any peer assessment exists. But as now we want it to be always skipped, the argument ``submission_uuid`` is not being used anymore.

Reply:
Makes sense, you can keep it in this case. Sorry for the false alarm.